ImplementationGuide/markdown/ReleaseNotes.md

```diff
@@ -6,8 +6,8 @@ Die erste Ziffer X bezeichnet ein Major-Release und regelt die Gültigkeit von R
 
 Version: 3.0.0-rc1
 
-- tbd.
-
+* ISiKVersicherungsverhaeltnisSelbstzahler
+  * Organisation als möglicher 'payor' hinzugefügt 
```

Comment:
ggf. auch ein Halbsatz, warum das erforderlich war (es gibt auch Konstellationen, in denen die Zahlung direkt über die PKV erfolgt)